src/scripts/main.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+const mainImage = document.getElementById('largeImg');
+const list = document.getElementById('thumbs');
+const links = document.body.querySelectorAll('.list-item__link');
+
+links.forEach(link => {
+  link.onclick = function() {
+    return false;
+  };
+});
+
+// eslint-disable-next-line no-shadow
+list.addEventListener('click', function(event) {
```

Comment:
```suggestion
list.addEventListener('click', (event) => {
```
It's better to use arrow function as a callback